build.gradle.kts

```diff
@@ -27,6 +27,7 @@ dependencies {
     implementation("org.conscrypt:conscrypt-openjdk-uber:2.5.2") {
         because("contains library code for all platforms")
     }
+    implementation("org.bouncycastle:bcutil-jdk15on:1.70")
```

Comment:
does this require java15+?

Reply:
I believe it's java 1.5 and up. 
https://www.bouncycastle.org/latest_releases.html

Reply:
lol! okay

Reply:
https://search.maven.org/artifact/org.bouncycastle/bcutil-jdk18on/1.71/jar

Reply:
done